src/DesignCompile/Elaboration_test.cpp

```diff
@@ -474,12 +475,14 @@ endmodule
       const std::string& name = cassign->Lhs()->VpiName();
       if (name == "p1") {
         // Val is 1, but it has a signed typespec (Meaning negative bin number)
-        EXPECT_EQ(val, 1);
+        EXPECT_EQ(val, 3);
```

Comment:
why did this value change ?

Reply:
Previous value was incorrect as unsigned where partially supported so far

Reply:
Now they get expended based on LHS size